appcues/src/main/java/com/appcues/debugger/ui/CaptureConfirmDialog.kt

```diff
@@ -0,0 +1,220 @@
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(16.dp),
+    ) {
+        Image(
+            painter = rememberAsyncImagePainter(capture.screenshot),
+            contentDescription = stringResource(id = string.appcues_screen_capture_image_description),
+            modifier = Modifier
+                .height(375.dp)
+                .border(1.dp, AppcuesColors.CaptureImageBorder),
+            contentScale = ContentScale.Fit,
+        )
+        OutlinedTextField(
+            value = text.value,
+            singleLine = true,
+            onValueChange = { text.value = it },
+            label = { Text(text = stringResource(id = string.appcues_screen_capture_text_input_label)) },
+            colors = TextFieldDefaults.outlinedTextFieldColors(
+                backgroundColor = Color.Transparent,
+                focusedBorderColor = AppcuesColors.CaptureTextInputBorder,
+                focusedLabelColor = AppcuesColors.ShadyNeonBlue,
+                unfocusedBorderColor = AppcuesColors.CaptureTextInputBorder,
+                unfocusedLabelColor = AppcuesColors.ShadyNeonBlue,
+            )
+        )
+        Row {
+            CaptureButton(
+                text = stringResource(id = string.appcues_screen_capture_cancel),
+                textColor = AppcuesColors.ShadyNeonBlue,
+                modifier = Modifier
+                    .height(40.dp)
+                    .background(AppcuesColors.DebuggerBackground)
+                    .border(1.dp, AppcuesColors.ShadyNeonBlue, RoundedCornerShape(6.dp))
+                    .clickable { debuggerViewModel.closeExpandedView() }
+            )
+            Spacer(modifier = Modifier.weight(1.0f))
+            CaptureButton(
+                text = stringResource(id = string.appcues_screen_capture_ok),
+                textColor = Color.White,
+                modifier = Modifier
+                    .height(40.dp)
+                    .background(
+                        Brush.horizontalGradient(listOf(AppcuesColors.ShadyNeonBlue, AppcuesColors.CaptureButtonGradientEnd)),
+                        RoundedCornerShape(6.dp)
+                    )
+                    .conditionalClickable(
+                        enabled = text.value.isEmpty().not(),
+                        onClick = {
+                            debuggerViewModel.onScreenCaptureConfirm(capture.copy(displayName = text.value))
+                        }
+                    )
+            )
+        }
+    }
+}
+
+private fun Modifier.conditionalClickable(enabled: Boolean, onClick: () -> Unit) = composed {
+    this.then(
+        if (enabled) {
+            Modifier.clickable {
+                onClick()
+            }
+        } else {
+            Modifier.alpha(ContentAlpha.disabled)
+        }
+    )
+}
+
+@Composable
+private fun CaptureButton(text: String, textColor: Color, modifier: Modifier) {
```

Comment:
Modifier is usually the first parameter, and also includes default value == Modifier.
```

CaptureButton(
    modifier: Modifier = Modifier
    .
    .
    .
)
```